starboard/android/shared/player_components_factory.cc

```diff
@@ -44,7 +44,11 @@ bool PlayerComponents::Factory::OutputModeSupported(
       return true;
     }
     DrmSystem* android_drm_system = static_cast<DrmSystem*>(drm_system);
-    return !android_drm_system->require_secured_decoder();
+    bool is_widevine_l1 = android_drm_system->require_secured_decoder();
+    SB_LOG_IF(ERROR, is_widevine_l1)
```

Comment:
This should be an INFO, as returning false from this function doesn't necessary lead to an error.

Reply:
Done.